pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/RocksdbMetadataStore.java

```diff
@@ -233,12 +240,15 @@ private RocksdbMetadataStore(String metadataURL, MetadataStoreConfig metadataSto
             throw new MetadataStoreException("Fail to create RocksDB file directory", e);
         }
 
+        readonly = "true".equalsIgnoreCase(
```

Comment:
I am not sure if it's a good idea to use system property to set up this config. It's not easy for user to know this config.

Reply:
My first impl version is using rocksdb config file (metadata config file) and the `rocksdb config file` is more used to work for rocksdb core, so i change it to system property for this config.

Do you have any suggestion?